### PR TITLE
PSBT Field

### PR DESCRIPTION
**Pull request by tirth2004**
Added a field for PSBT testing which calls the `signPsbt` method to handle PSBT signing requests.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -117,5 +117,8 @@
         }
       }
     }
+  },
+  "cli": {
+    "analytics": "55353286-fe67-4561-add5-413898aee3dd"
```
Comment (sondreb) [inline angular.json]: Remove this

**Review by sondreb — CHANGES_REQUESTED** [review]
Thanks for this, please remove that analytics key then it's fine!

**Comment by tirth2004** [comment]
@sondreb done

**Review by sondreb — APPROVED** [review]
Thanks!

**Pull request merged**